src/pages/validator/amplifier/verifier-onboarding.mdx

```diff
@@ -8,45 +8,106 @@ The Axelar Virtual Machine (AVM) and Amplifier are currently under active develo
+</tab-item>
+
+</tabs>
+
+## Add `ampd` to your `PATH`
+
+This will allow you to begin commands with `ampd`.
+
+1. Add a symbolic link named `ampd` to the `ampd` binary on your machine:
 
     ```bash
-    docker pull haiyizxx/tofnd:latest
-    docker run -p 50051:50051 --env MNEMONIC_CMD=auto --env NOPASSWORD=true -v tofnd:/.tofnd haiyizxx/tofnd:latest
+    sudo ln -s <BINARY NAME> /usr/local/bin/ampd
     ```
-    
-    Leave this process running in the background, and perform additional commands in a new terminal window or shell.
 
-    
-1. Download [`ampd`](https://github.com/axelarnetwork/axelar-amplifier/tree/main/ampd#worker-onboarding), the off-chain daemon responsible for voting and signing with the amplifier protocol. 
+1. Run the following to add an alias called `ampd` to the binary:
+
+    ```bash
+    alias ampd=<BINARY PATH>
+    ```
+
+1. Make the binary executable:
+
+    ```bash
+    chmod a+x ./ampd-darwin-arm64-v0.4.0
+    ```
 
-    We recommend downloading the `ampd` [binary](https://github.com/axelarnetwork/axelar-amplifier/releases/tag/ampd-v0.4.0) ([Linux](https://github.com/axelarnetwork/axelar-amplifier/releases/download/ampd-v0.4.0/ampd-linux-amd64-v0.4.0), [Apple Silicon Mac](https://github.com/axelarnetwork/axelar-amplifier/releases/download/ampd-v0.4.0/ampd-darwin-arm64-v0.4.0), [Intel Mac](https://github.com/axelarnetwork/axelar-amplifier/releases/download/ampd-v0.4.0/ampd-darwin-amd64-v0.4.0)). All instructions in this tutorial will assume you have a binary in your `PATH` named `ampd`.
+1. Restart your terminal.
 
-        - You can add the `ampd` binary to your path with `sudo ln -s <BINARY NAME> /usr/local/bin/ampd`.
-        - Alternatively, you can set an alias with `alias ampd=<BINARY PATH>` and restart your terminal. 
-        
-    You should make the binary executable with `chmod a+x ./ampd-darwin-arm64-v0.4.0`, customizing the binary name to match the one you downloaded.
+All instructions in this tutorial will assume you have a binary in your `PATH` named `ampd`. If you don't want to add `ampd` to your `PATH`, you can refer to the binary directly in every command in this tutorial, replacing `ampd` with the `ampd` version that corresponds with your machine.
 
-    If you don't want to add it to your path, you can refer to the binary directly in every command in this tutorial, such as `~/ampd-linux-amd64-v04.0 <command>`.
+For example, to determine the verifier address without the `ampd` alias on an Apple Silicon Mac, you would run:
 
-    If you get a security warning while trying to run `ampd` on Mac, you can disable the [gatekeeper](https://support.apple.com/guide/security/gatekeeper-and-runtime-protection-sec5599b66df/web) for a single binary with `sudo xattr -d com.apple.quarantine ./ampd-darwin-arm64-v0.4.0` or disable gatekeeper globally with `sudo spctl --master-disable`.
+```bash
+~/ampd-darwin-arm64-v0.4.0 verifier-address
```

Comment:
This needs to be configured per type (source, docker, binary)

Reply:
eg when I run commands with docker, I'll need to:
`docker run axelarnet/axelar-ampd:v0.4.0 verifier-address`